src/pages/Nominate/Active/index.tsx

```diff
@@ -79,15 +75,6 @@ export const Active = () => {
                   {t('nominate.start_nominating')}
                   <OpenHelpIcon helpKey="Nominations" />
                 </h3>
-                <div>
-                  <ButtonPrimary
-                    text={t('nominate.nominate')}
-                    iconLeft={faChevronCircleRight}
-                    iconTransform="grow-1"
-                    disabled={targets.length === 0 || inSetup() || isSyncing}
-                    onClick={() => openModalWith('Nominate', {}, 'small')}
-                  />
-                </div>
```

Comment:
Why are we removing this modal to nominate ?

Reply:
Polkadot's idea is that you select multiple validators and then click on "nominate" to commit the set of validators. This was the button to do it. Now on this screen, each validator has it's own "Nominate" button.